flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -144,6 +145,21 @@ public String toString() {
 			'}';
 	}
 
+	@Override
+	public boolean equals(Object o){
```

Comment:
We don't need to add `equals` method for `StateTtlConfig`, if you want to compare an instance with `StateTtlConfig.DISABLED`, you can just do it like `stateTtlConfig == StateTtlConfig.DISABLED`

Reply:
Got it, I will remove these code